site/static/examples/dra/gpu-quota-setup.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "default-flavor"
+---
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: DynamicResourceAllocationConfig
+metadata:
+  name: default
+  namespace: kueue-system
+spec:
+  resources:
+  - name: gpu.example.com
```

Comment:
it would be better if you use a different name for the resources vs the deviceClass so it's clear that what other CRs are actually referernce when they put "gpu.example.com" as the value.